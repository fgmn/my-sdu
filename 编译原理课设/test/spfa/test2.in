30
50
0
14
21
0
4
2
0
18
26
0
14
2
0
10
25
0
8
29
1
18
6
2
17
15
2
16
7
3
17
17
3
10
6
3
27
16
4
7
2
4
14
14
4
17
23
4
8
1
5
9
17
6
12
6
6
10
23
6
19
28
6
28
8
7
12
13
7
24
22
7
11
21
8
18
2
9
14
9
10
10
3
10
10
14
10
18
1
11
26
23
12
23
20
13
21
18
13
27
19
14
22
13
15
18
5
16
22
30
16
18
24
16
18
19
17
24
6
18
27
25
18
28
18
18
20
28
19
28
26
20
29
21
20
21
6
22
27
7
24
27
5
24
28
30
25
27
19
27
29
1
