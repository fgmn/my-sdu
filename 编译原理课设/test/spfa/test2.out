0
590558003
590558003
590558003
2
590558003
590558003
4
3
590558003
25
25
17
590558003
2
590558003
590558003
25
5
590558003
33
39
15
37
26
590558003
48
22
23
23
